### PR TITLE
Added spectral file to apply general oas 3 rules

### PR DESCRIPTION
**Pull request by ruvyas**
### Changes
___
* Added a `.spectral.yaml` file with general oas 3 rulesets recommended by @decarufe

### CODE REVIEW

**Review by Zav — APPROVED** [review]

**Comment by wagreniergsoft** [comment]
All good, but for future PR that security team asks to be a reviewer please wait our approval before merging :) You can add me as a reviewer.

**Pull request merged**
